chia/full_node/mempool_manager.py

```diff
@@ -173,18 +153,39 @@ def always(mm: MempoolManager, mi: MempoolItem) -> bool:
                     removals.extend(item.removals)
                     additions.extend(item.additions)
                 else:
-                    broke_from_inner_loop = True
-                    break
-        if len(spend_bundles) > 0:
-            log.info(
-                f"Cumulative cost of block (real cost should be less) {cost_sum}. Proportion "
-                f"full: {cost_sum / self.constants.MAX_BLOCK_COST_CLVM}"
-            )
-            agg = SpendBundle.aggregate(spend_bundles)
-            return agg, additions, removals
-        else:
+                    return (spend_bundles, uint64(cost_sum), additions, removals)
```

Comment:
this is a nit-pick: if you invert the if-statement and return, you don't need an else-block, and the code on line 150-154 no longer need to be indented. I think code generally reads easier with early-exits and fewer levels of indentation.